casper/src/main/scala/coop/rchain/casper/blocks/proposer/Proposer.scala

```diff
@@ -164,9 +165,7 @@ object Proposer {
         // TODO: temp this is done after validation
 //        MultiParentCasper.handleValidBlock(b) >>
         // broadcast hash to peers
-        CommUtil[F].sendBlockHash(b.blockHash, b.sender) >>
-        // Publish event
-        EventPublisher[F].publish(MultiParentCasper.createdEvent(b))
```

Comment:
Do you want to remove `EventPublisher` completely?

Reply:
Yes, no trace of it should be left. If you find something it means I didn't see it.